lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,15 +1,18 @@
-import React from 'react';
+import React from "react";
+import PropTypes from "prop-types";
 
 const Tab = props => {
   /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
       if they match, the className should be: 'tab active-tab', 
       if it is not it should just be 'tab'*/
+  console.log(props);
   return (
     <div
-      className={''}
+      className={`tab ${props.selected ? "active-tab" : ""}`}
```

Comment:
Hey Novina, you don't have `props.selected` you have `props.selectedTab`

Reply:
Thanks Samar. This has now been resolved. The console is now showing the correct data in the selectedTab.